src/Endpoints/ProductphotosEndpoint.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace JacobDeKeizer\Ccv\Endpoints;
+
+use JacobDeKeizer\Ccv\Exceptions\CcvShopException;
+use JacobDeKeizer\Ccv\Models\Productphotos as Models;
+use JacobDeKeizer\Ccv\Models\Products\Resource\Productphotos;
+
+class ProductphotosEndpoint extends BaseEndpoint
+{
+    /**
+     * @throws CcvShopException
+     */
+    public function get(int $id): Productphotos
+    {
+        $result = $this->doRequest('GET', 'productphotos/' . $id);
+
+        return Productphotos::fromArray($result);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function delete(int $id): void
+    {
+        $this->doRequest('DELETE', 'productphotos/' . $id);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function getAllForProduct(int $productId): Models\Collection\Productphotos
+    {
+        $result = $this->doRequest('GET', 'products/' . $productId . '/productphotos');
+
+        return Models\Collection\Productphotos::fromArray($result);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function create(int $productId, Models\Productphotos\Post $productphoto, bool $onlyFilledProperties = true): void
+    {
+        $this->doRequest(
+            'POST',
+            'products/' . $productId . '/productphotos',
+            $productphoto->toArray($onlyFilledProperties)
+        );
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function update(int $id, Models\Productphotos\Patch $productphoto, bool $onlyFilledProperties = true): void
+    {
+        $this->doRequest('PATCH', 'productphotos/' . $id, $productphoto->toArray($onlyFilledProperties));
+    }
+
+    /**
+     * Replace a product photo collection of an existing product.
+     * All existing photos will be deleted. With an empty collection you can achieve a DELETE ALL product photos.
+     *
+     * @throws CcvShopException
+     */
+    public function replace(
+        int $productId,
+        Models\Collection\Productphotos $productphotosCollection,
```

Comment:
You can use src/Models/Productphotos/Productphotos/Put.php here, because this is the put method. When you use this model, you can remove the array_map and use toArray on the Put object.

Reply:
Yes, stupid that I didn't see that